homeassistant/components/solarlog/const.py

```diff
@@ -165,11 +165,12 @@ class SolarLogSensorEntityDescription(
         state_class=STATE_CLASS_TOTAL_INCREASING,
     ),
     SolarLogSensorEntityDescription(
-        key="total_power",
+        key="installed_peak_power",
```

Comment:
I don't think we need to change the key since that will change the unique id.

Reply:
If we really want to change key, we should migrate existing entity entries to the new unique id.

Reply:
You are right, I've undone the key-change.